### PR TITLE
Data Access Controls exercise

### PR DESCRIPTION
**Pull request by lboeman**
First stab at this, It needs to contrast with the other users, but I was wondering if it's too 'in the weeds' for the length of the workshop or what we actually want to demonstrate.

### CODE REVIEW

**Comment by wholmgren** [comment]
A couple of sections felt a bit technical, but I don't think it's too in the weeds overall. Write the remaining sections and then let's discuss/walkthrough.

File: data_access_control.md [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+in the [Site section of the Data Model](https://solarforecastarbiter.org/datamodel/#site) document.
+
+  - Again, the other permissions allow the Forecast user to view the role and permissions, as these permissions are usually
+reserved for administrators.
+
+### The Permission Page
+Click on the 'Read Plant X observation values' permission. The permission page shows some information about the permission
+as well as listing the objects that it allows its action on. In this case, all of the observations made by Utility X at Power
+Plant X are listed. If Utility X no longer wanted to share data from a portion of these observations, they would simply need
+to remove those observations from the permissions, although editting roles and permissions through the dashboard is not yet
```
Comment (wholmgren) [inline data_access_control.md]: to remove those observations from the permissions. Support for editing roles and permissions through the dashboard will be implemented soon.

File: data_access_control.md [hunk 2]
```diff
@@ -0,0 +1,117 @@
+# View Data Access Controls
+
+The Solar Forecast Arbiter uses roles and permissions to control access to data and resources within the system.
```
[the remark this answers is not in the document]
Comment (lboeman) [reply data_access_control.md]: Perhaps data and metadata is all that needs to be said here in the data access control document, but the roles and permissions also control access to users, roles and permissions on the site as well maybe even trials and reports.

File: data_access_control.md [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+A permission defines an action that can be made on an resource or group of resources, e.g. 'create a site' or
+'read forecast forecast A'. A role is a collection of permissions that can be assigned to a user. A user can
+be assigned multiple roles.
+
+Here we use the following scenario to demonstrate how data access controls function in the Solar Forecast Arbiter.
+
+A utility company, _Utility X_ has a power plant, Plant X with some irradiance and power measurements. They want
+to share Site information and measurements with _Forecaster A_ and _Forecaster B_.
+Forecasters A and B want to provide power forecasts for Power Plant X that are only visible to their own users
+and to users in Utility X. The forecasters have also made forecasts for a site, 'Ashland OR', which is not visible
```
[the remark this answers is not in the document]
Comment (lboeman) [reply data_access_control.md]: The site is not visible to others, but I will allow the Utility to read the site as well. Perhaps the site should be changed to the more generic 'Weather Station 1'. This is the Ashland OR site I set up as a demo site.

[the remark this answers is not in the document]
Comment (wholmgren) [reply data_access_control.md]: Let's avoid the potential source of confusion by changing the name to something generic like you've suggested.

**Pull request merged**